packages/dev-middleware/src/inspector-proxy/Device.js

```diff
@@ -71,7 +78,7 @@ export default class Device {
   #deviceSocket: WS;
 
   // Stores the most recent listing of device's pages, keyed by the `id` field.
-  #pages: $ReadOnlyMap<string, Page>;
+  #pages: $ReadOnlyMap<string, PageWithMiddleware>;
```

Comment:
Any Flow pro's around that may have a better idea for this one? It seems that this causes some issues in both the `this.#pageHasCapability` and the `this.#newLegacyReactNativePage` functions due to Flow not accepting that `Page` is a subset of `PageWithMiddleware`.

Basically, it doesn't accept `Page & {middleware: ?DeviceMessageMiddleware}` as being also a `Page` type. And `{...Page,...}` causes some trouble with the `this.#lastConnectedLegacyReactNativePage` property.

Reply:
For now, I fixed it by allowing both `Page | PageWithMiddleware` _exact_ object types. But, might be nicer to allow inexact in utility functions/properties in `Device`.